explorer/src/Pos/Explorer/Web/Server.hs

```diff
@@ -827,8 +827,8 @@ cAddrToAddr cAddr@(CAddress rawAddrText) =
     in case mDecodedBase64 of
         Just addr -> do
             -- the decoded address can be both the RSCoin address and the Cardano address.
-            -- * RSCoin address == 32 bytes
-            -- * Cardano address >= 34 bytes
+            -- 1. RSCoin address == 32 bytes
```

Comment:
According to [this section](http://haskell-haddock.readthedocs.io/en/latest/markup.html#itemized-and-enumerated-lists) the removed syntax was compatible -- do you see any immediate advantage into switching to the proposed one?

Reply:
-- * : raises a haddock error.  -- * is fine for top level documentation, not for comment inside the code.

Reply:
Oh, TIL I guess, thanks!